### PR TITLE
fix(rollingOperatorUpgradePipeline): remove withEnv and make it use routines

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/SPcGhigl/2670-deprecate-withenv-in-the-upgrade-pipeline
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by dkropachev** [comment]
Tested - https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/dk-upgrade-scylla-operator-minikube/42/

**Comment by dkropachev** [comment]
Test faield as planned (due to the s-o fails on major upgrade), cleanup succeeded

**Comment by dkropachev** [comment]
Tested on s-o pipeline: https://jenkins.scylladb.com/view/master/job/scylla-master/job/rolling-upgrade/job/major-upgrade-scylla-operator-gke/25/
and on longevity pipeline: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/dmitry-byo-longevity/428/

**Review by enaydanov — CHANGES_REQUESTED** [review]

File: vars/getJobTimeouts.groovy [hunk 1]
```diff
@@ -7,10 +7,19 @@ List<Integer> call(Map params, String region){
...
+
+    if [[ -n "${params.scylla_mgmt_agent_version ? params.scylla_mgmt_agent_version : ''}" ]] ; then
+        export SCT_SCYLLA_MGMT_AGENT_VERSION=${params.scylla_mgmt_agent_version}
+    fi
+    ./docker/env/hydra.sh output-conf -b "${params.backend}"
     """
-    def testDuration = sh(script: cmd, returnStdout: true).trim()
-    testDuration = testDuration.toInteger()
+    def testData = sh(script: cmd, returnStdout: true).trim()
+    println(testData)H
```
[the remark this answers is not in the document]
Comment (dkropachev) [reply vars/getJobTimeouts.groovy]: removed

**Review by enaydanov — APPROVED** [review]

**Pull request merged**